package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "discord.js": "^14.7.1",
     "dotenv": "^16.3.1",
+    "ms": "^2.1.3",
```

Comment:
What does this do?

Reply:
Hi Soraalam1, this adds the 'ms' package to the project, allowing for conversions of date/time formats such as "1h23m22s" or "12:00pm" to be converted into milliseconds for our bot's convenience.